tests/order.py

```diff
@@ -92,5 +106,39 @@ def test_remove_product_from_order(self):
         self.assertEqual(len(json_response["lineitems"]), 0)
 
     # TODO: Complete order by adding payment type
+    def test_complete_order_by_adding_payment(self):
+
+        # Add product to order
+        url = "/cart"
+        data = {"product_id": 1}
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
+        response = self.client.post(url, data, format="json")
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # complete an order using cart ID
+        url = "/cart/1/complete"
+        data = {"payment_id": self.payment_id}
+
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
+        response = self.client.put(url, data, format="json")
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # Perform GET request to verify the payment type is assigned
+        url = "/orders"
```

Comment:
When a cart is completed, it is moved to `/orders`. If we used `/cart` it would be empty because it no longer has items in it after payment is applied.